api/src/main/java/com/exadel/frs/core/trainservice/system/global/ImageProperties.java

```diff
@@ -21,14 +21,16 @@
 import javax.validation.constraints.Size;
 import lombok.Data;
 import org.springframework.boot.context.properties.ConfigurationProperties;
-import org.springframework.context.annotation.Configuration;
+import org.springframework.stereotype.Component;
 
-@Configuration
+@Component
 @ConfigurationProperties(prefix = "image")
 @Data
 public class ImageProperties {
 
     @NotNull
     @Size(min = 1)
     private final List types;
+
+    private boolean save;
```

Comment:
Could you rename property to `saveImagesToDB`? I know that it's longer, but in one year we will forget what does `save` mean